scala/private/rule_impls.bzl

```diff
@@ -1351,10 +1347,9 @@ def _jacoco_offline_instrument(ctx, input_jar):
 
     ctx.actions.run(
         mnemonic = "JacocoInstrumenter",
-        inputs = [in_out_pair[0] for in_out_pair in in_out_pairs] + worker_inputs,
+        inputs = [in_out_pair[0] for in_out_pair in in_out_pairs],
         outputs = [in_out_pair[1] for in_out_pair in in_out_pairs],
-        executable = ctx.attr._code_coverage_instrumentation_worker.files_to_run.executable,
-        input_manifests = worker_input_manifests,
+        executable = ctx.attr._code_coverage_instrumentation_worker.files_to_run,
```

Comment:
This seems fine to me. Do you (@johnynek?) know if we've re-enabled the coverage tests in this repo? They were disabled, but I recall seeing a PR to fix/reenable them for current Bazel versions.